crates/bevy_ui/src/node_bundles.rs

```diff
@@ -18,9 +18,12 @@ use bevy_sprite::TextureAtlas;
 use bevy_text::{BreakLineOn, JustifyText, Text, TextLayoutInfo, TextSection, TextStyle};
 use bevy_transform::prelude::{GlobalTransform, Transform};
 
-/// The basic UI node
+/// The basic UI node.
 ///
-/// Useful as a container for a variety of child nodes.
+/// [`Node`] component and required components to make node work.
```

Comment:
This wording doesn't really make sense. Maybe something like this would be better:
```suggestion
/// Contains the [`Node`] component and its required components.
```

Reply:
the node component doesn't have required components.

it's the set of components required to make a container in the ui

Reply:
Rephrased. Hope it's better.